fastly/provider.go

```diff
@@ -14,8 +15,14 @@ import (
 // any API requests made by the provider.
 const TerraformProviderProductUserAgent = "terraform-provider-fastly"
 
+// This value can be set to allow terraform output to display sensitive info
+var DisplaySensitiveFields bool
+
 // Provider returns a *schema.Provider.
 func Provider() *schema.Provider {
+
+	DisplaySensitiveFields = os.Getenv("FASTLY_TERRAFORM_PROVIDER_DISPLAY_SENSITIVE_FIELDS") == "true"
```

Comment:
I would recommend to change this to: `FASTLY_TF_DISPLAY_SENSITIVE_FIELDS` to keep it a little bit shorter but I'll keep this up to you and Kevin.